desktop/core/src/desktop/lib/botserver/views_tests.py

```diff
@@ -185,11 +187,11 @@ def test_handle_gist_link(self):
             "user": {
               "is_bot": False,
               "profile": {
-                "email": "test@example.com"
+                "email": "test@{domain}".format(domain=self.host_domain)
```

Comment:
It is great to test with `'testserver.gethue.com'` and keep it that way, but the true domain should be `'gethue.com'`, so should email here should be `test@gethue.com`?

Reply:
To extract the domain from the hostname, using [tldextract](https://pypi.org/project/tldextract/) @romainr

Reply:
Could we use a native Python lib instead? Or even just rsplit on .

Reply:
Using split or native library like urllib.parse can be flaky for some hostnames types  like test.mytest.gethue.com and test.gethue.co.in)

Any other way to extract the domain from host name or do we know some more details something like the suffixes (.com or .co.in only types) possible etc @romainr ?

Reply:
Currently just checking for domain like `gethue.com` and updated UTs